compiler_opt/rl/local_data_collector.py

```diff
@@ -34,7 +33,7 @@ class LocalDataCollector(data_collector.DataCollector):
 
   def __init__(
       self,
-      module_specs: List[corpus.ModuleSpec],
+      corp: corpus.Corpus,
```

Comment:
corpus

Reply:
Yep, conflicts with the package. but corpora is technically plural

Reply:
there are several files both importing corpus and using a Corpus; only doing the renamed import in this file is probably gonna be more confusing, renaming everything might be better?